### PR TITLE
Add keyboard shortcuts overview

### PR DESCRIPTION
**Pull request by romanroibu**
This PR adds keyboard shortcuts overview tables to Pupil Capture and Pupil Player docs.

Depends on [this PR](https://github.com/pupil-labs/pupil/pull/2129) being merged.

---

[ClickUp Task](https://app.clickup.com/t/gq33tk)

### CODE REVIEW

**Review by papr — APPROVED** [review]

File: src/core/software/pupil-player.md [hunk 1]
```diff
@@ -37,6 +37,19 @@ The Player window is the main control center for `Pupil Player`. It displays vid
 1. **Menu**: This area contains settings and contextual information for each plugin.
 1. **Sidebar**: This area contains clickable buttons for each plugin. System plugins are loaded in the top and user added plugins are added below the horizontal separator.
 
+### Keyboard Shortcuts
+
+| Keyboard Shortcut   | Description                                      |
+|:--------------------|:-------------------------------------------------|
+| `<space>`           | Play and pause video                             |
+| `<arrow left>`      | Step to previous frame / Decrease playback speed |
```
Comment (willpatera) [inline src/core/software/pupil-player.md:45]: can you clarify how this keyboard shortcut can be both stepping and increasing/decreasing playback speed?

**Review by willpatera — COMMENTED** [review]
Please see my question re playback speed in Player. Once the other change is merged/bundle deployed @papr can merge.

File: src/core/software/pupil-player.md (hunk 1 above)
Comment (papr) [reply src/core/software/pupil-player.md:45]: The frame skipping and playback-speed changes are explained directly above the table. Do you think this is too much out of context and requires to be re-iterated?

Comment (papr) [reply src/core/software/pupil-player.md:45]: What do you think about this footnote solution?
Screenshot 2021-04-13 at 09 18 58

**Comment by willpatera** [comment]
Looks good to me :+1:

**Pull request merged**